python-sdk/tutorials/automl-with-azureml/classification-bank-marketing-all-features/auto-ml-classification-bank-marketing-all-features.ipynb

```diff
@@ -228,8 +228,8 @@
     "n_missing_samples = int(np.floor(data.shape[0] * missing_rate))\n",
     "missing_samples = np.hstack(\n",
     "    (\n",
-    "        np.zeros(data.shape[0] - n_missing_samples, dtype=np.bool),\n",
-    "        np.ones(n_missing_samples, dtype=np.bool),\n",
+    "        np.zeros(data.shape[0] - n_missing_samples, dtype=np.bool_),\n",
+    "        np.ones(n_missing_samples, dtype=np.bool_),\n",
```

Comment:
Just for my understanding - do we need to use np.bool_(scalar type) here or should we use bool?

Reply:
updated to bool!